README.md

```diff
@@ -1,24 +1,151 @@
-# README
+# Hello Rails Back-End
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<a name="readme-top"></a>
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
```

Comment:
- [ ] It would be nice if you fix the FAQ link because it is broken 😨 Kindly fix it or remove it (If you don't have any dedicated section for this link) to make your readme more descriptive and professional 👍

Reply:
HI @HAMMAS-SALEEM ,

Thanks for the feedback. I have removed the broken link.

Best Regards,
K. Inoulen Shanongba